wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -75,77 +43,72 @@ export default function ShopModalView(props: ShopModelViewProps) {
         onChange={props.handleImageSelection}
       />
       <div className="col-6">
-        {/* Test 1 */}
         <WavyModalHeader image={props.headerPictureUrl || defaultHeaderPicture} onClose={props.onClose}>
-          <div className={style.buttonOpenPictureUpload}>
-            <Button
-              className="btn-lg"
-              onClick={props.openPicUploader}
-            >
-              add diving events here
-              {/* <span className="hide-sm">{screenData.DiveSite.addSightingButton}</span> */}
-            </Button>
-          </div>
-
           <div className={style.buttonImageUpload}>
-            <ButtonIcon
+            {props?.isPartnerAccount && (
+              <ButtonIcon
               icon={<Icon name="camera-plus" />}
               className="btn-lg"
-              onClick={() => fileUploaderRef?.current?.click?.()}
-            />
+              onClick={() => {}}
+              />
+            )}  
           </div>
         </WavyModalHeader>
         <div className="ml-6">
           <div className="stack-4">
             <div>
               <div className="d-flex">
-                <h1 className="mb-0">{props?.shopModelName}</h1>
-                <div>
-                  <Icon
-                    name="flag"
-                    fill="maroon"
-                    width="30px"
-                    onClick={() => window.location = `mailto:DiveGo2022@gmail.com?subject=Reporting%20issue%20with%20Dive%20Site:%20"${selectedDiveSite.SiteName}"%20at%20Latitude:%20${selectedDiveSite.Latitude}%20Longitude:%20${selectedDiveSite.Longitude}&body=Type%20of%20issue:%0D%0A%0D%0A%0D%0A%0D%0A1)%20Dive%20site%20name%20not%20correct%0D%0A%0D%0A(Please%20provide%20correct%20dive%20site%20name%20and%20we%20will%20correct%20the%20record)%0D%0A%0D%0A%0D%0A%0D%0A2)%20Dive%20site%20GPS%20coordinates%20are%20not%20correct%0D%0A%0D%0A(Please%20provide%20a%20correct%20latitude%20and%20longitude%20and%20we%20will%20update%20the%20record)`}
-                  />
-                </div>
+                <h1 className="mb-0">{props?.diveShop.orgName}</h1>
```

Comment:
Ts is complaining here you'll need another "?" after diveShop to satisfy Ts, when Ts complains that it can possibly  be null its asking for a check